src/execution/compiler/codegen.cpp

```diff
@@ -85,6 +85,12 @@ ast::Expr *CodeGen::Const64(int64_t val) const {
   return expr;
 }
 
+ast::Expr *CodeGen::ConstU32(uint32_t val) const {
```

Comment:
As alluded to above, this really isn't something to address in the ANALYZE PR. If others agree with me on the type naming consistency issue, I can open an issue for this to be refactored later. Otherwise, we can just leave it alone without any negative consequences.